beta/src/pages/learn/sharing-state-between-components.md

```diff
@@ -1,31 +1,31 @@
 ---
-title: Sharing State Between Components
+title: कौम्पोनॅन्ट्स के बीच state शेयरिंग
 ---
 
 <Intro>
 
-Sometimes, you want the state of two components to always change together. To do it, remove state from both of them, move it to their closest common parent, and then pass it down to them via props. This is known as *lifting state up,* and it's one of the most common things you will do writing React code.
+कभी-कभी आप चाहेंगे की दो कौम्पोनॅन्ट्स की state हमेशा एक साथ बदले। ऐसा करने के लिए पहले उन दोनों में से state को निकल दें, फिर उन्हें मूव कर के उनके निकटतम कॉमन पैरेंट के पास ले जाएँ, और फिर props के माध्यम से उनमें पास कर दें। इसे _"लिफ्टिंग state अप"_ कहा जाता है, और React कोड लिखे जाते समय, किया जाने वाला यह सबसे प्रचलित तरीका है।
 
 </Intro>
 
 <YouWillLearn>
 
-- How to share state between components by lifting it up
-- What are controlled and uncontrolled components
+- लिफ्टिंग के द्वारा कौम्पोनॅन्ट्स के बीच state कैसे शेयर की जाये
+- कंट्रोल्ड और अन-कंट्रोल्ड कौम्पोनॅन्ट्स क्या होते हैं
 
 </YouWillLearn>
 
-## Lifting state up by example {/*lifting-state-up-by-example*/}
+## लिफ्टिंग state अप, उदाहरण के माध्यम से {/*lifting-state-up-by-example*/}
 
-In this example, a parent `Accordion` component renders two separate `Panel`s:
+इस उदाहरण में एक पैरेंट `Accordion` कोम्पोनेंट दो विभिन्न `Panel`s को रेंडर करता है:
```

Comment:
`कोम्पोनेंट` => `कौम्पोनॅन्ट`

we use this translation of component to keep it consistent. For more info on this you can also refer #2